bin/yauml

```diff
@@ -19,9 +19,10 @@
 #                  http://www.gnu.org/licenses/                                     *
 #************************************************************************************
 
-import sys, yaml, getopt, re, subprocess
+import os, sys, yaml, getopt, re, subprocess
 
-template_filename = '../templates/default.dot'
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+TEMPLATE_FILENAME = CURRENT_DIR + '/../templates/default.dot'
```

Comment:
This can't work with the way that yauml script is installed. It is installed to `/usr/bin/` and templates will be installed to `/usr/share/yauml/`. yauml won't find the template then. This is a standard way of installing files on a UNIX-like platform.

Providing a relative path to the script is not safe after installation. I think that absolute paths are always preferred.